armory/scenarios/cifar10.py

```diff
@@ -0,0 +1,97 @@
+"""
+CIFAR10 scenario evaluation
+"""
+
+import logging
+
+from tqdm import tqdm
+
+from armory.utils.config_loading import (
+    load_dataset,
+    load_model,
+    load_attack,
+    load_defense,
+)
+from armory.utils import metrics
+from armory.scenarios.base import Scenario
+
+logger = logging.getLogger(__name__)
+
+
+class Cifar10(Scenario):
+    def _evaluate(self, config: dict) -> dict:
+        """
+        Evaluate the config and return a results dict
+        """
+
+        model_config = config["model"]
+        classifier, preprocessing_fn = load_model(model_config)
+
+        if not model_config["weights_file"]:
+            classifier.set_learning_phase(True)
+            logger.info(
```

Comment:
Do we think a null weight's file is inutitive enough for users to know it will train a model. I lean toward yes, but we've seen flags like `model_status` and `pretrained` etc. so wondering if we should think about that a bit. (Not gaiting this PR though)

Reply:
I think it's intuitive enough; however, maybe we want to a different flag for whether to train it. I personally don't like the "model_status" and "pretrained" flags. A wider discussion of specific flags seems in order after we have the other branches merged.